### PR TITLE
Added number scroller for randomization

### PR DESCRIPTION
**Pull request by WChrisK**

### CODE REVIEW

File: app/models/assignment.rb [hunk 1]
```diff
@@ -7,6 +7,9 @@ class Assignment < ActiveRecord::Base
     rubric: 'rubric'
   }
 
+  MIN_PEER_REVIEWS_PER_PERSON = 1
+  MAX_PEER_REVIEWS_PER_PERSON = 5
```
Comment (WChrisK) [inline app/models/assignment.rb]: Do you think these are needed? I don't know if in the future we want to constrain the admins (though currently they can just enter whatever value in the input field regardless of the above) and remove this, or keep it and use it for something later on?

File: app/assets/stylesheets/peer_reviews.scss [hunk 2]
```diff
@@ -41,3 +41,11 @@
   padding-left: 19em;
   padding-right: 33em;
 }
+
+.peer-review-scrollbar-div {
+  padding-bottom: 8px;
+}
```
Comment (WChrisK) [inline app/assets/stylesheets/peer_reviews.scss]: Is the `-div` redundant? (should be removed?)

File: app/models/assignment.rb (hunk 1 above)
Comment (david-yz-liu) [inline app/models/assignment.rb]: Yeah, I don't think we need a max. Keep the min, though.

File: app/assets/stylesheets/peer_reviews.scss (hunk 2 above)
Comment (david-yz-liu) [inline app/assets/stylesheets/peer_reviews.scss]: It's alright

File: app/views/peer_reviews/_peer_reviews_new_manager.js.jsx.erb [hunk 3]
```diff
@@ -96,6 +96,11 @@
       return (
         <div>
           <ErrorDiv error={this.state.error} />
+          <div className="peer-review-scrollbar-div">
+            <%= j raw t('peer_review.number_per_group') %>: <input type="number"
```
Comment (david-yz-liu) [inline app/views/peer_reviews/_peer_reviews_new_manager.js.jsx.erb]: I think you want to give the input a name so that it's passed in `params`

Comment (WChrisK) [inline app/views/peer_reviews/_peer_reviews_new_manager.js.jsx.erb]: Currently it's not part of any form that's submitted, I was hoping to use JavaScript to pull out the value when a user presses the random button and transmit that via ajax.
Assuming there's not a better way and the above is to be done, can the name be left off then?

Comment (david-yz-liu) [inline app/views/peer_reviews/_peer_reviews_new_manager.js.jsx.erb]: You can give it an id then

File: app/assets/stylesheets/peer_reviews.scss (hunk 2 above)
Comment (david-yz-liu) [inline app/assets/stylesheets/peer_reviews.scss]: Looking at it again though, I don't know if `scrollbar` is the best choice of noun

File: app/assets/stylesheets/peer_reviews.scss [hunk 4]
```diff
@@ -41,3 +41,11 @@
   padding-left: 19em;
   padding-right: 33em;
 }
+
+.peer-review-amount-spinner {
+  padding-bottom: 8px;
+}
```
Comment (WChrisK) [inline app/assets/stylesheets/peer_reviews.scss]: Looks like spinner was the name I was looking for in the end.

**Comment by david-yz-liu** [comment]
Thanks, @WChrisK!

**Pull request merged**